david-minaya/tab-group
# Add the index page of the extension

This page shows the different tabs groups that the user has created. To open this page the user must click the button "Open page" in the popup of the extension. From this page, the user cans open the tabs groups doing click in the option "Open" of the tabs groups.

### Q&A

To test this code:

1. Open the index page of the extension doing click in the button "Open page" of the popup.
2. Open a new tab group doing click in the option "Open" of one of the tabs groups.
3. If all work fine, then all is ok.

If you don't see the tabs groups on the index page, that is because you don't have tabs groups created. To create tabs groups use the popup of the extension.
